docs/en/application.md

```diff
@@ -154,4 +179,151 @@ After the `Application` resource is applied to Kubernetes cluster, the KubeVela
 
 > TBD: the revision names and labels for resource instances are currently work in progress.
 
-> TBD: a demo for kubectl apply above Application CR and show full detailed underlying resources.
+## Verify generated resources
+Apply these definitions and application yaml above.
+
```

Comment:
1. get the application and see the status
2. Just show the application by `kubectl get deploy` is fine, no more details needed.